src/pe_reports/metrics.py

```diff
@@ -12,25 +22,571 @@ def __init__(self, start_date, end_date, org_uid):
+            .reset_index()
+        )
+
+        if len(verif_vulns) > 0:
+            verif_vulns["count"] = verif_vulns["ip"].str.split("  ").str.len()
+            verifVulns = verif_vulns["count"].sum()
+
+        else:
+            verifVulns = 0
+
+        return verifVulns
+
+    def unverified_cve(self):
+        """Return top 15 unverified cves and their counts."""
+        insecure_df = self.insecure_df
+        unverif_df = insecure_df[insecure_df["type"] != "Insecure Protocol"]
+        unverif_df = unverif_df.copy()
+        unverif_df["potential_vulns"] = (
+            unverif_df["potential_vulns"].sort_values().apply(lambda x: sorted(x))
+        )
+        unverif_df["potential_vulns"] = unverif_df["potential_vulns"].astype("str")
+        unverif_df = (
+            unverif_df[["potential_vulns", "ip"]]
+            .drop_duplicates(keep="first")
+            .reset_index(drop=True)
+        )
+        unverif_df["count"] = unverif_df["potential_vulns"].str.split(",").str.len()
+        unverif_df = unverif_df[["ip", "count"]]
+        unverif_df = unverif_df.sort_values(by=["count"], ascending=False)
+        unverif_df = unverif_df[:15].reset_index(drop=True)
+        return unverif_df
+
+    def unverified_vuln_count(self):
+        """Return the count of ips with unverified vulnerabilities."""
+        insecure_df = self.insecure_df
+        unverif_df = insecure_df[insecure_df["type"] != "Insecure Protocol"]
+        unverif_df = unverif_df.copy()
+        unverif_df["potential_vulns"] = (
+            unverif_df["potential_vulns"].sort_values().apply(lambda x: sorted(x))
+        )
+        unverif_df["potential_vulns"] = unverif_df["potential_vulns"].astype("str")
+        unverif_df = (
+            unverif_df[["potential_vulns", "ip"]]
+            .drop_duplicates(keep="first")
+            .reset_index(drop=True)
+        )
+
+        return len(unverif_df.index)
+
+    def verif_vulns(self):
+        """Return a dataframe with each CVE, the associated ips and the affected ports."""
+        vulns_df = self.vulns_df
+        verif_vulns = (
+            vulns_df[["cve", "ip", "port"]]
+            .groupby("cve")
+            .agg(lambda x: "  ".join(set(x)))
+            .reset_index()
+        )
+        return verif_vulns
+
+    def verif_vulns_summary(self):
+        """Return summary dataframe for verified vulns."""
+        vulns_df = self.vulns_df
+        verif_vulns_summary = (
+            vulns_df[["cve", "ip", "port", "summary"]]
+            .groupby("cve")
+            .agg(lambda x: "  ".join(set(x)))
+            .reset_index()
+        )
+
+        verif_vulns_summary = verif_vulns_summary.rename(
+            columns={
+                "cve": "CVE",
+                "ip": "IP",
+                "port": "Port",
+                "summary": "Summary",
+            }
+        )
+        return verif_vulns_summary
+
+
+class Cyber_Six:
+    """Dark Web and Cyber Six Data Class."""
```

Comment:
Remove unnecessary capitalization:
```suggestion
    """Dark web and Cyber Six data class."""
```